articles/systems/04-teleporting-system.md

```diff
@@ -1,23 +1,53 @@
 
-1. Step 1
-2. Step 2
+![](/images/Configuration/TeleportSystem/TeleportSystemProfile.png)
+
+## Teleport System platform configuration
+
+The Teleport system for the XRTK allows for different implementations to be utilized for different runtime or build platforms. Each platform provided by the XRTK includes a default Teleport system configuration for that platform which can be customized if you wish:
+
+![](/images/Configuration/TeleportSystem/TeleportSystemSettings.png)
+
+The configuration holds:
+
+* Teleport Provider - Select the system to be used to transport the user through the environment when a Teleport event is received.
+* Teleport Action - The Input Action that the teleport system will used to enact a Teleportation event.  **Please Note** that some action provide additional data and capabilities, such as a Dual-Axis action will enable Teleportation with rotation and the ability to force rotate the view.
+* Teleport Data Provider configuration - The data provider list available to the Teleport system, primarily to validate whether a teleport location is valid or not.
+
+> For more detail on the [Platforms](08-platform-system.md) that are available to configure, see the [XRTK Platform System](08-platform-system.md) documentation.
+
+## Mixed Reality Teleport data providers
+
+The Teleport system allows for different validation methods to be used based on the capabilities of the target platform, these validation profiles are defined in this data providers list, indicating the teleport validation for the platform and the specific platform the system is applicable for.
+
+![](/images/Configuration/TeleportSystem/TeleportSystemDataProviders.png)
+
+## Teleport Validation Profile
+
+The Teleport Validation profile allows you to configure the requirements for a successful teleport, checking the user has selected a valid target by the physics properties or the selection such as physics layers, exclusion layers, max angle and the distance from the origin.  These are used by the teleport system to ensure a teleport event is safe (within the bounds of the project) to perform.
+
+![](/images/Configuration/TeleportSystem/TeleportValidationOptions.png)
+
+## Further notes
+
+If `None` is selected as the Teleport Provider for the project, then by default, the XRTK will perform an "Instant" teleport when a teleport is successful.  We also provide a "Fading Teleport" method in the SDK for use and reference should you wish to build your own Teleport method.
+
+For the most part, you should not need to alter the Teleport system configuration unless required.
 
 ---
 
 ### Related Articles
 
-* [Article 1]()
-* [Article 2]()
-* [Article 3]()
+* [XRTK Platform System](08-platform-framework.md)
```

Comment:
I'm not sure platform stuff needs to be linked at the bottom of all these systems. It's not really related